@types/APIClients.ts

```diff
@@ -103,6 +103,8 @@ type RetryPolicyConfiguration = {
     retryInterval?: number;
     exponentialBackoffBase?: number;
     exponentialBackoffScale?: number;
+    statusCodes?: number[];
+    retryMethods?: string[];
```

Comment:
Since we're tidying this - worth getting the naming here consistent? So everything is either pretended with `retry` or nothing is?